### ISSUE TITLE
Causes crash on Mining profession

### ISSUE THREAD

**Issue by ArztVielfrass** [ExcessiveAmountsOfZombies/Professions#6]
Playing multiplayer with 3 people, while working as a miner, mining a block causes a crash.

**Comment by ExcessiveAmountsOfZombies** [comment]
MC version, Mod version and fabric/forge?

**Comment by ArztVielfrass** [comment]
A crash report is probably more helpful: [---- Minecraft Crash Report ----.txt](https://github.com/ExcessiveAmountsOfZombies/Professions/files/9388872/----.Minecraft.Crash.Report.----.txt)

I am using Professions as part of a [modpack](https://www.curseforge.com/minecraft/modpacks/medieval-mc-fabric)
I used CurseForge to install the modpack.

Replication:

1. Open world
2. Set professions hotkey to Right Menu
3. Join Miner
4. Mine Stone Block with Stone Pickaxe
> Crash

I tried doing this on singleplayer and it crashes all the same, some occupations doesn't immediately crash after fulfilling a reward condition like Logger and it seems like every occupation causes a crash.
